features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/Workflow/APIStateChange.jsx

```diff
@@ -113,8 +119,8 @@ function ListLabels() {
             setData(LocalData);
         })
             .catch((e) => {
-                Alert.error('Unable to fetch data ' + e);
-                console.error('Unable to fetch data ' + e);
+                Alert.error('Unable to fetch data. ', e.message);
```

Comment:
Internationalize the static message

Reply:
Why are we showing a static error and the error from backend as well?

Reply:
fixed with 9c884f8